src/controllers/QueryController.ts

```diff
@@ -115,15 +115,21 @@ export class QueryController extends RootComponent {
    * Create a new query controller
    * @param element
    * @param options
+   * @param usageAnalytics @deprecated
```

Comment:
I think you could explain "why" this is deprecated, and that internally the class will resolve usage analytics through the search interface instead.